install/roles/server/tasks/uninstall_podman_containers.yml

```diff
@@ -23,7 +23,7 @@
   register: podman_qpc_ps_raw
   ignore_errors: yes
 
-- name: Set variable indicating whether QPC server podman container found
+- name: Set variable indicating whether quipucords server podman container found
```

Comment:
Change to `Set variable indicating whether the Quipucords server podman container is present`